frigate/object_detection.py

```diff
@@ -54,19 +52,12 @@ def __init__(
         else:
             self.input_transform = None
 
-        if det_type == DetectorTypeEnum.edgetpu:
-            self.detect_api = EdgeTpuTfl(
-                det_device=det_device, model_config=model_config
-            )
-        elif det_type == DetectorTypeEnum.openvino:
-            self.detect_api = OvDetector(
-                det_device=det_device, model_config=model_config
-            )
-        else:
+        if det_type == DetectorTypeEnum.cpu:
```

Comment:
Looking at it, this CPU warning should move into the cpu detector? Then we won't need any detector-specific conditions here.

Reply:
the warning is moved into the create_detector function now